imgui_ros/Projector projector
---
string message  # ""
bool success  # False
